e3sm_diags/plot/aerosol_aeronet_plot.py

```diff
@@ -22,7 +21,50 @@
 ]
 # Border padding relative to subplot axes for saving individual panels
 # (left, bottom, right, top) in page coordinates.
-BORDER_PADDING = (-0.06, -0.03, 0.13, 0.03)
+BORDER_PADDING_COLORMAP = (-0.06, 0.25, 0.13, 0.25)
+BORDER_PADDING_SCATTER = (-0.08, -0.04, 0.15, 0.04)
+
+
+def _save_plot_aerosol_aeronet(fig, parameter):
+    """Save aerosol_aeronet plots with different border padding for each panel."""
+    import os
+
+    from matplotlib.transforms import Bbox
+
+    from e3sm_diags.driver.utils.io import _get_output_dir
+
+    # Save the main plot
+    for f in parameter.output_format:
+        f = f.lower().split(".")[-1]
+        fnm = os.path.join(
+            _get_output_dir(parameter),
+            parameter.output_file + "." + f,
+        )
+        plt.savefig(fnm)
+        logger.info(f"Plot saved in: {fnm}")
+
+    # Save individual subplots with different border padding
+    border_paddings = [BORDER_PADDING_COLORMAP, BORDER_PADDING_SCATTER]
+
+    for f in parameter.output_format_subplot:
+        fnm = os.path.join(
+            _get_output_dir(parameter),
+            parameter.output_file,
+        )
+        page = fig.get_size_inches()
+
+        for idx, (panel, border_padding) in enumerate(zip(PANEL_CFG, border_paddings)):
+            # Extent of subplot
+            subpage = np.array(panel).reshape(2, 2)
+            subpage[1, :] = subpage[0, :] + subpage[1, :]
+            subpage = subpage + np.array(border_padding).reshape(2, 2)
+            subpage_list = list(((subpage) * page).flatten())
+            extent = Bbox.from_extents(*subpage_list)
+
+            # Save subplot
+            fname = fnm + ".%i." % idx + f
+            plt.savefig(fname, bbox_inches=extent)
+            logger.info(f"Sub-plot saved in: {fname}")
```

Comment:
Much of the code in `_save_plot_aerosol_aeronet()` is identical to `utils.plot._save_plot()` (below).
https://github.com/E3SM-Project/e3sm_diags/blob/436456117022b59550c1d8188490e36d12f78662/e3sm_diags/plot/utils.py#L60-L124

I think the only difference  `for` loop on line 56. I suggest we extend `_save_plot()` (or break it up into smaller functions) to support this difference in order to keep the code maintainable.